### PR TITLE
Remove all uses of -force-single-frontend-invocation

### PR DESCRIPTION
**Pull request by davezarzycki**
The `-force-single-frontend-invocation` flag predates WMO and is now an alias for `-whole-module-optimization`. We should use the latter and let the former fade into history.

### CODE REVIEW

**Comment by davezarzycki** [comment]
@swift-ci please smoke test

File: include/swift/Option/Options.td [hunk 1]
```diff
@@ -935,11 +935,6 @@ def disable_batch_mode : Flag<["-"], "disable-batch-mode">,
 def wmo : Flag<["-"], "wmo">, Alias<whole_module_optimization>,
   Flags<[FrontendOption, NoInteractiveOption, HelpHidden]>;
 
-def force_single_frontend_invocation :
```
Comment (harlanhaskins) [inline include/swift/Option/Options.td:938]: I’m fine with removing the flag but let’s leave this option here as an alias for -whole-module-optimization. We have existing clients passing -force-single-frontend-invocation and we can’t break them.

**Review by harlanhaskins — CHANGES_REQUESTED** [review]

**Review by CodaFi — CHANGES_REQUESTED** [review]
Thanks for the cleanup! I agree with Harlan's assessment: Let's leave behind an alias for now. You can even warn in the Driver about the flag being an alias. Any non-experimental flags in the driver are supposed to be considered part of a stable interface.

**Comment by compnerd** [comment]
I think that the option alias is a requirement ... that is a driver flag.  Only the frontend flags can be removed, driver flags need to be maintained for compatibility.

**Comment by compnerd** [comment]
@swift-ci please test Windows platform

**Comment by davezarzycki** [comment]
I added the old flag alias back.

@swift-ci pleas smoke test

**Comment by davezarzycki** [comment]
@swift-ci please test Windows platform

**Review by harlanhaskins — APPROVED** [review]

**Comment by davezarzycki** [comment]
@swift-ci please smoke test

**Review by CodaFi — APPROVED** [review]

**Pull request merged**